packages/eslint-plugin/src/rules/naming-convention-utils/format.ts

```diff
@@ -17,26 +17,26 @@ https://gist.github.com/mathiasbynens/6334847
 function isPascalCase(name: string): boolean {
   return (
     name.length === 0 ||
-    (name[0] === name[0].toUpperCase() && !name.includes('_'))
```

Comment:
I actually think that the previous form is clearer for this specifica case.
[It's also substantially faster (not that it matters at the this speed... but it is twice as fast - ~1.2B ops/s vs ~2.4B ops/s)](https://jsbench.me/buljbfvcn4/1).

I wonder if we should add an option to skip this case from the rule?

Reply:
Agreed. The main 👎 on this change in my head was that the rule is suggesting _longer_ code... #7140